### PR TITLE
[MULTIARCH-1443] Add fips support for IBM zSystems and IBM Power

### PR DESCRIPTION
**Pull request by SNiemann15**
Version(s): 4.13 + 

Issue: 
- Z: https://issues.redhat.com/browse/MULTIARCH-1443
- P: https://issues.redhat.com/browse/MULTIARCH-1442

Link to docs preview:
- [Support for FIPS cryptography](https://57725--docspreview.netlify.app/openshift-enterprise/latest/installing/installing-fips.html)
- [Sample install-config.yaml file for IBM Power](https://57725--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_ibm_power/installing-ibm-power.html#installation-bare-metal-config-yaml_installing-ibm-power)
- [Sample install-config.yaml file for IBM zSystems](https://57725--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_ibm_z/installing-ibm-z.html#installation-bare-metal-config-yaml_installing-ibm-z)

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by SNiemann15** [comment]
@manojnkumar @holgwolf Can you please review. 

fyi @alishaIBM  @jaypoulz

**Comment by holgwolf** [comment]
lgtm

File: modules/installation-configuration-parameters.adoc [hunk 1]
```diff
@@ -687,7 +687,7 @@ ifndef::openshift-origin[]
 |Enable or disable FIPS mode. The default is `false` (disabled). If FIPS mode is enabled, the {op-system-first} machines that {product-title} runs on bypass the default Kubernetes cryptography suite and use the cryptography modules that are provided with {op-system} instead.
 [IMPORTANT]
 ====
-The use of FIPS Validated / Modules in Process cryptographic libraries is only supported on {product-title} deployments on the `x86_64` architecture.
+TThe use of FIPS Validated / Modules in Process cryptographic libraries is only supported on {product-title} deployments on `x86_64`, `ppc64le`, and `s390x` architectures.
```
Comment (jaypoulz) [inline modules/installation-configuration-parameters.adoc:690]: double `T` on this line

**Comment by jaypoulz** [comment]
Just the extra letter I found, otherwise looks good!

**Comment by SNiemann15** [comment]
@mburke5678 Can you please remove the `branch/enterprise-4.12` label. 
I've created  a new PR for the backport to 4.12 because it requires CM: https://github.com/openshift/openshift-docs/pull/58575

**Review by kelbrown20 — COMMENTED** [review]
LGTM!

**Pull request merged**